MAINTAIN.md

```diff
@@ -143,6 +143,7 @@ These dependencies are "vendored" (inlined), we must update the sources manually
     * Run `scripts/gen_lsp.lua` to update.
 * `src/bit.c`: only for PUC lua: port of `require'bit'` from luajit https://bitop.luajit.org/
 * [treesitter parsers](https://github.com/neovim/neovim/blob/fcc24e43e0b5f9d801a01ff2b8f78ce8c16dd551/cmake.deps/CMakeLists.txt#L197-L210)
+* `runtime/lua/coxpcall.lua`: coxpcall (only needed for PUC lua, builtin to luajit)
```

Comment:
Is there a canonical upstream URL where this lives? That is helpful to mention here and/or in the source file.